src/blocks/BulletPointBlock/BulletPointBlock.tsx

```diff
@@ -0,0 +1,217 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {
+  ContentWrapper,
+  getMinimalColorStyles,
+  LAPTOP_BP_UP,
+  MOBILE_BP_DOWN,
+  MOBILE_BP_UP,
+  SectionWrapper,
+  TABLET_BP_UP,
+} from 'components/blockHelpers'
+import { DeferredImage } from 'components/DeferredImage'
+import { getStoryblokImage, Image } from 'utils/storyblok'
+import {
+  BrandPivotBaseBlockProps,
+  MarkdownHtmlComponent,
+  MinimalColorComponent,
+} from 'blocks/BaseBlockProps'
+import { FontSizes, Heading } from 'components/Heading/Heading'
+
+const BulletPointSectionWrapper = styled(SectionWrapper)`
+  overflow-x: hidden;
+`
+
+const InnerWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  min-width: 100%;
+
+  ${MOBILE_BP_DOWN} {
+    padding-left: 8px;
+    padding-right: 8px;
+  }
+
+  ${MOBILE_BP_UP} {
+    margin-left: -1.5rem;
+  }
+
+  ${LAPTOP_BP_UP} {
+    margin-left: -3.5rem;
+  }
+`
+
+const BulletPointTitle = styled(Heading)`
+  margin-bottom: 1rem;
+`
+
+const BulletPoint = styled.div<{
+  alignCenter: boolean
+  bulletPointLayout: boolean
+  columns: number
+}>`
+  display: flex;
+  flex-direction: ${(props) => (props.bulletPointLayout ? 'row' : 'column')};
+  width: 100%;
+  margin-top: 1.25rem;
+  margin-bottom: 1.25rem;
+
+  ${(props) =>
+    props.alignCenter &&
+    `
+      align-items: center;
+      text-align: center;
+    `}
+
+  ${MOBILE_BP_UP} {
+    flex-direction: column;
+    width: calc(50% - 1.5rem);
+    margin-left: 1.5rem;
+  }
+
+  ${TABLET_BP_UP} {
+    ${(props) =>
+      props.columns === 3 &&
+      `
+        width: calc((1 / 3 * 100%) - 1.5rem)
+      `}
+  }
+
+  ${LAPTOP_BP_UP} {
+    width: ${(props) => `calc((1 / ${props.columns} * 100%) - 3.5rem)`};
+    margin-left: 3.5rem;
+  }
+`
+
+const BulletPointHead = styled.div<{
```

Comment:
This name isn't super clear to me 🤔 Is it only used to wrap the image? Then maybe the name could be something that explains that? Or what's the "Head"? 🙈

Reply:
I agree, `BulletPointHeader` would make more sense IMO :)